### PR TITLE
Add signUp with useForm  and header with toggle theme

### LINKED ISSUE #18: Create Sign Up

### PR DESCRIPTION
**Pull request by AipovAlexander**
**1. Task name**  
Create header and footer #15
Create Sign Up #18

**2. Issue link**  
[header](https://github.com/MaxLisyanskiy/strada-pet/issues/15)
[signUp](https://github.com/MaxLisyanskiy/strada-pet/issues/18)

**3. Tasks**

- [x] Create header
- [x] Create signUp with useForm

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ const App = () => {
           <Route path="*" element={<NotFoundPage />} />
         </Route>
       </Routes>
+      //{' '}
```
Comment (MaxLisyanskiy) [inline src/App.tsx]: Удалить

**Review by MaxLisyanskiy — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (AipovAlexander) [reply src/App.tsx]: Не заметил, поправил

**Review by MaxLisyanskiy — APPROVED** [review]

File: src/components/header.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'space-between',
+        }}
+      >
+        <Menu mode="horizontal">
+          <Typography
+            style={{
+              margin: '0 0 0 10px',
```
Comment (ArtemVasilencko) [inline src/components/header.tsx]: marginLeft думаю получше будет написать

File: src/components/header.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            <Button type="primary">
+              <Link to="/home">To Home Page</Link>
+            </Button>
+          </Menu.Item>
+          <Menu.Item key="signin">
+            <Button>
+              <Link to="/signin">Sign In</Link>
+            </Button>
+          </Menu.Item>
+          <Menu.Item key="sinup">
```
Comment (ArtemVasilencko) [inline src/components/header.tsx]: тут в кее sinup a не signup

**Review by MaxLisyanskiy — APPROVED** [review]

**Pull request merged**